### DISCUSSION TITLE
How to get have options to remove edge, defender and provisioned packages?

### DISCUSSION THREAD

**Discussion by FrostedVolcano** [ChrisTitusTech/winutil#2615] [General]
From the recent stream on Titus Tech Talk channel I learned that the options to remove edge, defender and provisioned packages are going to be removed because of various reasons. But I still want to have those options. I know removing edge later is a better process but I want to remove provisioned packages and removing defender later causes some issues with windows. 
So, how can I add those options for myself. I don't know coding but I can understand a little bit and can make, so I hope someone can at least tell me how I can do it.

**Comment by ChrisTitusTech** [comment]
This was added back edge can be removed
